### ISSUE TITLE
Possibly update capitalization 

### ISSUE THREAD

**Issue by SteveLLamb** [SMPTE/eg428-23#2]
From feedback via email
- “projection mask” in section 4 is expected to start with lower case?
- Figures’ title seem to have every words start with capital letter, except some like Figures 6, 14, 15, 16
- Table 1 also does not have each word start with capital letter, while Pierre seems to have adopted all first letter as capital letter
 
Generally should look at capitalization for Tables, Figures, and Terms, based on guidelines in https://github.com/SMPTE/html-pub once complete (expected to be done prior to this document's PCD close).

**Comment by akiseino** [comment]
I think it's almost time to fix the capitalization issue. And I checked the guideline and I'll fix like below;
Figures' title: Capitalize only for first letter of first word.
Table's title: Capitalize for first letter of all words except preposition and conjunction.
Regarding Terms, I checked several STs and found no uniform notation. (ST 2086:2014 has no capital words. ST 2098-5:2018, 
ST 2065-3:2020 and ST 430-14:2022 have capital first letter of all words ) . Therefore, I'd like to capitalize the first letter of all words, which was more commonly used. In addition, the guidelines do not seem to specify about it.
